### ISSUE TITLE
When I click confirm on an item

### ISSUE THREAD

**Issue by Geczy** [pomkos/money_requesting#45]
It hides all my submissions and only shows the one item
I expect to see them all
![image](https://user-images.githubusercontent.com/1036968/119229151-28804b00-badc-11eb-8b85-9bc7e073b7ef.png)

**Comment by pomkos** [comment]
Yeah this was supposed to be for letting user review their selection.

New proposal: show the selection in a placeholder with the whole df below it

```
placeholder = st.empty()
{code}
with placeholder:
    st.write(user_selection_df)
```
